wasm/validator/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "iroha_validator"
+
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
+# Enables debugging tools such as `dbg()` and `DebugUnwrapExt`
+debug = ["iroha_wasm/debug"]
```

Comment:
seems unnecessary?

Reply:
it doesn't do anything except active feature of another crate. User should just activate that feature directly on `iroha_wasm` crate

Reply:
> it doesn't do anything except active feature of another crate. User should just activate that feature directly on iroha_wasm crate

How if user will only link to `iroha_validator`?

Reply:
I see, that it worked for `client/tests/integration/smartcontracts`. But that just becasue it has `iroha_wasm` as a direct depenency as well

Reply:
> I see, that it worked for `client/tests/integration/smartcontracts`. But that just becasue it has `iroha_wasm` as a direct depenency as well

my suggestion was that whenever user needs `debug` feature they would bring the `iroha_wasm` as explicit dependency considering that it's always there as implicit dependency. I'm not sure if this is preferable or not, maybe someone else would care to comment

Reply:
had these two crates `iroha_wasm` and `wasm_validator` been truly independent I wouldn't mention this

Reply:
I'm not sure it's a good thing, but yes, we can wait for someone else

Reply:
@appetrosyan @Erigara @QuentinI thoughts?

Reply:
I'm in favor of leaving it be. Doesn't cost us anything and gives more intuitive UX.